### DISCUSSION TITLE
Not scare users away from NeuroDesk!

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#136] [General]
Right now, if users stumbles upon the "More information on NeuroDesk" page here:
https://neurodesk.github.io/docs/neurodesk
they might be scared away by the long list of requirements.

Below the line "Quickstart: https://github.com/NeuroDesk/vnm/"
We should have something like 
'"Requirements: 
- having Docker installed
- having Web Browser installed"

We should remember that not everybody are familiar with containers (yet), and might not understand that running docker means that the host OS and its configuration are redundant.

Also, might be nice if the hyperlinks in https://neurodesk.github.io/docs/neurodesk would work.

**Comment by aswinnarayanan** [comment]
Hi @civier 

I didn't quite understand the "long list"; there is only three requirements for neurodesk under [https://neurodesk.github.io/docs/neurodesk](https://neurodesk.github.io/docs/neurodesk)
Lmod has now been moved to an optional section

The VNM requirements have space requirements added due to feedback from an actual user raised in [this issue](https://github.com/NeuroDesk/vnm/issues/105)

VNM requirements have been reworded to look cleaner.
The hyperlinks should be fixed now

**Comment by civier** [comment]
Yes. You are right. Three is not too bad, but still longer than a single
requirement. Unfortunately, this can make the difference to some users
(especially if they just want to check NeuroDesk out).

The bigger issue is that the page is confusing -- because the requirements
and the install instructions for the VNM are not listed, users might assume
that the requirements and install instructions below are also relevant to
the VNM. This is why I suggested to list the requirements for the VNM (only
one) on this page as well.

In any case -- i think it will be good to think what is the landing page we
want users to see first, and what info to put there in order to motivate
users to give NueroDesk a try. It is just too easy!

(Another question: what are the instructions for Mac? I don't see them on
the page)

Best,
Oren

--
Oren Civier
Neuroimaging Informatics Fellow
Swinburne Neuroimaging | National Imaging Facility
https://anif.org.au/team/oren-civier/
https://www.swinburne.edu.au/neuroimaging

Swinburne University of Technology
Hawthorn Campus, Advanced Technologies Centre, Room 914
+61 3 9214 4628 | +61 4 3101 6603
***@***.*** (Email/Teams video call)
https://twitter.com/orencivier

**Reply by aswinnarayanan** [reply]
Hi @civier. The landing page has been updated to clarify the difference between VNM and the Neurodesk installer.
The Mac instructions are on the VNM page.

Have any users reported that the requirements are too many for the Neurodesk installer?
I couldn't find any issues raised by users claiming this

**Reply by civier** [reply]
Thanks Aswin. Maybe it's just me with the requirements. We should really ask prospective users what they think (but don't expect people to complain. If somebody finds a software too difficult to install, most probably that she/he will simply look for another solution).

The changes you made look great, and now the website is much more welcoming.

Just two minor things:
1. You have a double "either" in this sentence:
Neurodesk can be run either using either VNM or the Neurodesk installer
2. Users that want to connect should open a thread in the discussions forum rather than the issues. I spoke about it with @stebo85 over email, and he wants the issues forum to be reserved to things that we actually work on.

Cheers,
Oren

**Reply by aswinnarayanan** [reply]
Typo is fixed
